Xamarin.Essentials/HapticFeedback/HapticFeedback.uwp.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Windows.Devices.Haptics;
+
+namespace Xamarin.Essentials
+{
+    public static partial class HapticFeedback
+    {
+        internal static bool IsSupported => true;
+
+        static async void PlatformExecute(HapticFeedbackType type)
+        {
+            try
+            {
+                if (Windows.Foundation.Metadata.ApiInformation.IsTypePresent(typeof(VibrationDevice).FullName) && await VibrationDevice.RequestAccessAsync() == VibrationAccessStatus.Allowed)
```

Comment:
I think you have to use the string. If the type is not present, then you can't get the typeof.

Reply:
ok